### ISSUE TITLE
Newtonsoft.Json 7.0.1 warning on PCL targeting Windows 8.1 + Windows Phone 8.1

### ISSUE THREAD

**Issue by nelsonghezzi** [JamesNK/Newtonsoft.Json#626]
Installing version 7.0.1 in a Portable Class Library that targets Windows 8.1 + Windows Phone 8.1 causes the proyect to emit multiple warnings with the folllowing message:

*Reference to type 'System.IConvertible' claims it is defined in 'c:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETPortable\v4.6\Profile\Profile32\mscorlib.dll', but it could not be found*

**Environment Information:**
Windows 8.1 Pro

Microsoft Visual Studio Ultimate 2013
Version 12.0.40629.00 Update 5
Microsoft .NET Framework
Version 4.5.51650

(.Net Framework 4.6 not installed)

**Comment by JamesNK** [comment]
You should be using the .NET 4.5 PCL build. It doesn't have IConvertible in it.

**Comment by nelsonghezzi** [comment]
I upgraded the package via Nuget from 6.0.8 to 7.0.1.

Previously, it was using the version of the assembly located at *portable-net40+sl5+wp80+win8+wpa81* (which was referenced automatically when the package was first installed).

After the package upgrade, it continued using the same profile. I also tested a clean install of the package, with that same profile being picked.

Now I've manually referenced the one you say, *portable-net45+wp80+win8+wpa81+dnxcore50*, which works.

Is this a "breaking" change in 7.x? Is Nuget missing something to select the correct assembly?

Thanks.
